cli/tsconfig.json

```diff
@@ -0,0 +1,15 @@
+{
+  "extends": "../common/tsconfig.json",
+  "compilerOptions": {
```

Comment:
Why is this different from the other projects?

Reply:
It was created by `oclif`
I've now removed most of it and it looks like
```json
{
  "extends": "../common/tsconfig.json",
  "compilerOptions": {
    "outDir": "lib",
    "rootDir": "src",
  },
  "include": [
    "src/**/*"
  ]
}
```
I'll check if I can change `lib` to `dist` easily and remove `include` and `rootDir` in future PRs

Reply:
If you defer this to a future PR please open an issue.

Reply:
#125